### PR TITLE
Feature/apply manifest for deploy commands

### PR DESCRIPTION
**Pull request by Anonycoders**

### CODE REVIEW

File: fandogh_cli/fandogh_client/__init__.py [hunk 1]
```diff
@@ -326,3 +305,50 @@ def deploy_manifest(manifest):
         raise get_exception(response)
     else:
         return response.json()
+
+
+def _generate_manifest_yaml(image, version, name, port, envs, hosts, internal, registry_secret, image_pull_policy,
+                            internal_ports):
+    yaml = dict()
```
Comment (s-soroosh) [inline fandogh_cli/fandogh_client/__init__.py]: manifest instead of yaml?

File: fandogh_cli/fandogh_client/__init__.py [hunk 2]
```diff
@@ -326,3 +305,50 @@ def deploy_manifest(manifest):
...
+        yaml['name'] = name
+
+    spec = dict()
+
+    if image:
+        spec['image'] = '{}:{}'.format(image, version)
+
+    env_lis = []
+    if version:
+        env_lis.append({'name': 'VERSION', 'value': version})
```
Comment (s-soroosh) [inline fandogh_cli/fandogh_client/__init__.py]: why is it needed?

File: fandogh_cli/managed_service_commands.py [hunk 3]
```diff
@@ -40,5 +41,26 @@ def help():
             click.echo("\t\t. {}:\t{}".format(parameter_name.ljust(20), description))
 
 
+def _generate_managed_manifest_yaml(name, version, config):
+    yaml = dict()
```
Comment (s-soroosh) [inline fandogh_cli/managed_service_commands.py]: manifest

File: fandogh_cli/fandogh_client/__init__.py (hunk 1 above)
Comment (Anonycoders) [reply fandogh_cli/fandogh_client/__init__.py]: agreed

File: fandogh_cli/fandogh_client/__init__.py (hunk 2 above)
Comment (Anonycoders) [reply fandogh_cli/fandogh_client/__init__.py]: VERSION in manifest is hard coded so we have to do it ourselves
but the check statement is redundant.

File: fandogh_cli/managed_service_commands.py [hunk 4]
```diff
@@ -40,5 +41,26 @@ def help():
...
+def _generate_managed_manifest(name, version, config):
+    manifest = dict()
+    manifest['kind'] = 'ManagedService'
+
+    spec = dict()
+    spec['service_name'] = name
+    spec['version'] = version
+
+    param_list = []
+    for param in config:
```
Comment (s-soroosh) [inline fandogh_cli/managed_service_commands.py]: use _parse_key_values

**Pull request merged**